BENEFITS.md

```diff
@@ -0,0 +1,17 @@
+
+## Benefits 
+* Advertisment of offering as "certified"
+* Use of Argo Logo
```

Comment:
Perhaps we should only allow use of the certification mark and not the official logo itself?
Note that the certification mark may essentially be the argo logo with some modifications.

Reply:
To me, this is a list of explicitly allowed things. It might be good to have a list of explicitly denied things.

Reply:
I agree with @edlee2121, we should have specific certification "badge", similar to that of Prometheus (https://prometheus.io/blog/2021/05/03/introducing-prometheus-conformance-program/)

As these would be exemptions from the CNCF trademark rules, I'm not sure if we need to explicitly deny things, as anything not explicitly allowed would resort back to the CNCF rules.

Reply:
Resolved in committee.